buildkite/pipeline.yml

```diff
@@ -0,0 +1,39 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  - &common_plugins
+    - &docker_plugin
+      docker#v3.8.0:
+        image: &ruby_version "ruby:3.2.2"
```

Comment:
There are a few times CI jobs failed due to [the Docker Hub rate limit](https://docs.docker.com/docker-hub/download-rate-limit/). We have since started using [AWS Public ECR images](https://gallery.ecr.aws/docker/library/ruby). I'd suggestion that we use `public.ecr.aws/docker/library/ruby:3.2.2` here too.

Reply:
Good to know, thanks 👍